src/bin/cargo/commands/run.rs

```diff
@@ -12,6 +12,7 @@ pub fn cli() -> Command {
         .arg_quiet()
         .arg(
             Arg::new("args")
+                .help("Arguments for the binary or example to run")
```

Comment:
Should these changes be reflected in `cargo help run`, etc? 

`rustc` and `rustdoc` do not acknowledge `args`.  `run` acknowledges `args`, indirectly, in the description but not in section for the options.

Reply:
Looking at it closer, I think the `args` in the description would have stood out more if the italics were rendered but something in my pipeline isn't (likely my terminal).